corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -489,11 +489,14 @@ def _format_last_sync_time(restore_user, sync_time=None):
     return ServerTime(sync_time).user_time(timezone).done().isoformat()
 
 
-def generate_rows_and_filters(report_data_cache, report_config, restore_user, row_to_element):
+def generate_rows_and_filters(
+    report_data_cache, report_config, restore_user, row_to_element, current_row_count=None
```

Comment:
Would it make sense to change the `current_row_count` default to `0` (zero) rather than `None`?